addons/l10n_it_edi/models/account_edi_format.py

```diff
@@ -212,14 +212,14 @@ def _l10n_it_goods_in_italy(self, invoice):
 
     def _l10n_it_document_type_mapping(self):
         return {
-            'TD01': dict(move_type='out_invoice', import_type='in_invoice'),
-            'TD04': dict(move_type='out_refund', import_type='in_refund'),
-            'TD07': dict(move_type='out_invoice', import_type='in_invoice', simplified=True),
-            'TD08': dict(move_type='out_refund', import_type='in_refund', simplified=True),
-            'TD09': dict(move_type='out_invoice', import_type='in_invoice', simplified=True),
-            'TD17': dict(move_type='in_invoice', import_type='out_invoice', self_invoice=True, services_or_goods="service"),
-            'TD18': dict(move_type='in_invoice', import_type='out_invoice', self_invoice=True, services_or_goods="consu", partner_in_eu=True),
-            'TD19': dict(move_type='in_invoice', import_type='out_invoice', self_invoice=True, services_or_goods="consu", goods_in_italy=True),
+            'TD01': dict(move_types=['out_invoice'], import_type='in_invoice'),
+            'TD04': dict(move_types=['out_refund'], import_type='in_refund'),
+            'TD07': dict(move_types=['out_invoice'], import_type='in_invoice', simplified=True),
+            'TD08': dict(move_types=['out_refund'], import_type='in_refund', simplified=True),
+            'TD09': dict(move_types=['out_invoice'], import_type='in_invoice', simplified=True),
+            'TD17': dict(move_types=['in_invoice', 'in_refund'], import_type='out_invoice', self_invoice=True, services_or_goods="service"),
+            'TD18': dict(move_types=['in_invoice', 'in_refund'], import_type='out_invoice', self_invoice=True, services_or_goods="consu", partner_in_eu=True),
+            'TD19': dict(move_types=['in_invoice', 'in_refund'], import_type='out_invoice', self_invoice=True, services_or_goods="consu", goods_in_italy=True),
```

Comment:
@lordkrandel Is this ok to you?

Reply:
I need to have 'in_refund' as a type for the reverse charge refund, but it uses the same document type as TD17,18,19 .. except with negative values

Reply:
Lgtm